stregsystem/templates/mail/deposit_manual.html

```diff
@@ -14,8 +14,8 @@
             ====================================================================<br>
             Hello {{ firstname }}!<br><br>
             We've had great trouble inserting {{ formatted_amount }} stregdollars on your account: "{{ username }}". <br><br>
-            This is due to you not you not writing your username correctly, and instead writing '{{ mobilepay_comment }}'. The poor TREOs had to take multiple minutes out of their day to insert your money manually.
-            If you want to utilise our automatic fill-up in future, you can write your username correctly in the MobilePay comment: '{{ username }}'
+            This is due to you not writing your username correctly, and instead writing '{{ mobilepay_comment }}'. The poor TREOs had to take multiple minutes out of their day to insert your money manually.
+            If you want to utilise our automatic fill-up in the future, you can write your username correctly in the MobilePay comment: '{{ username }}'

```

Comment:
```suggestion
            If you want to utilise our automatic fill-up in future, you can write your username correctly in the MobilePay comment: '{{ username }}'

```
This is correct and proper British English: https://english.stackexchange.com/questions/2005/do-in-future-and-in-the-future-imply-different-meanings